packages/flutter/lib/src/rendering/animated_size.dart

```diff
@@ -68,6 +73,38 @@ class RenderAnimatedSize extends RenderAligningShiftedBox {
     _animation.curve = value;
   }
 
+  /// The [TickerProvider] for the [AnimationController] that runs the animation.
+  TickerProvider get tickerProvider => _tickerProvider;
+  TickerProvider _tickerProvider;
+  set tickerProvider(TickerProvider value) {
+    assert(value != null);
+    if (value == _tickerProvider)
+      return;
+    AnimationController oldController = _controller;
+    assert(oldController != null);
+    _setupAnimations(tickerProvider, _controller.duration, _animation.curve);
+    if (oldController.isAnimating) {
+      _controller.forward(from: oldController.value);
```

Comment:
Won't this drop a frame?  The `oldController.value` is the value we had at the last tick but calling `forward` will cause the first frame of the new control (i.e., next frame) to have that value as well.

Instead, I think we need some way of changing the ticker provider for a running animation, presumably by handing the timebase off between the tickers so they don't repeat a frame.